src/NewTools-Debugger-Tests/StDebuggerActionModelTest.class.st

```diff
@@ -478,6 +500,33 @@ StDebuggerActionModelTest >> testProceedDebugSession [
 	self assert: result equals: 4
 ]
 
+{ #category : #tests }
+StDebuggerActionModelTest >> testRecompileMethodToInContextNotifyingUpdatesSourceCodeAndContext [
+
+	| oldStack contextChanged expectedNewStack rejectedFromOldStack |
+	contextChanged := debugActionModel topContext.
+
```

Comment:
I would like a comment that explains the structure of the stack => how does the stack evolve with the step until?